packages/flutter/lib/src/widgets/semantics_debugger.dart

```diff
@@ -173,7 +173,7 @@ class _SemanticsDebuggerState extends State<SemanticsDebugger> with WidgetsBindi
         child: Listener(
           onPointerDown: _handlePointerDown,
           behavior: HitTestBehavior.opaque,
-          child: IgnorePointer(
+          child: AbsorbPointer(
```

Comment:
This work because I didn't convert AbsorbPointer. Do we want to convert it?
The semantics debugger is one rare exception that we want the old behavior(ignore pointer events while leaving the subtree semantics untouched)